tools/update_assisted_installer_yaml.py

```diff
@@ -11,22 +12,56 @@
 args = parser.parse_args()
 
 
+def in_all_lists(el: str, lists: list[str]) -> bool:
+    for list in lists:
+        if el not in list:
+            return False
+    return True
+
+
+def find_first_common_element(lists: list[str]) -> str:
+    if len(lists) < 1:
+        raise IndexError("No lists provided. Cannot find common element")
+    for el in lists[0]:
+        if in_all_lists(el, lists[1:]):
+            return el
+    raise RuntimeError("Could not find common element")
+
+
+def get_ref_by_docker_image(images):
+    tag_pattern = "^latest-[a-f0-9]{40}$"
+    tag_prefix = "latest-"
+    sk = skopeo.Skopeo()
```

Comment:
can we change all names here to not include abbrev?
e.g. ``sk->skeopeo_client``, ``img->image``
it will make code more English-like